numba/tests/test_record_dtype.py

```diff
@@ -1047,12 +1044,11 @@ def test_py_argument_charseq(self):
 
         # compile
         rectype = numpy_support.from_dtype(recordwithcharseq)
-        cres = compile_isolated(pyfunc, (rectype[:], types.intp,
-                                         rectype.typeof('n')))
-        cfunc = cres.entry_point
+        sig = (rectype[::1], types.intp, types.Bytes(types.uint8, 1, 'C'))
+        cfunc = njit(sig)(pyfunc)
 
         for i in range(self.refsample1d.size):
-            chars = "{0}".format(hex(i + 10))
+            chars = bytes("{0}".format(hex(i + 10)), 'UTF-8')
```

Comment:
This change defeats the intention of the test, which is to test `CharSeq` as argument. Suggest revert then change 
 
```suggestion
        sig = (rectype[::1], types.intp, rectype.typeof('n'))
        cfunc = njit(sig)(pyfunc).overloads[sig].entry_point

        for i in range(self.refsample1d.size):
            chars = "{0}".format(hex(i + 10))
```

Reply:
Thanks, fixed in ce9f24e.